var/spack/repos/builtin/packages/py-deeptools/package.py

```diff
@@ -12,12 +12,13 @@ class PyDeeptools(PythonPackage):
 
     # The test suite and associated test data is missing in the pypi tarball.
```

Comment:
Is this comment still valid? Usually this means we chose GitHub because the PyPI tarball didn't work. Can you ensure that old versions still build properly?

Reply:
I've removed the patch as per the next comment - as you say we don't run any tests anyway. Have confirmed that the oldest version here (`@2.5.2`) installs fine from PyPi.